addons/account_edi/views/account_move_views.xml

```diff
@@ -87,32 +87,41 @@
                     <field name="edi_error_count" invisible="1" />
                     <div class="alert alert-info" role="alert" style="margin-bottom:0px;"
                         attrs="{'invisible': ['|', ('edi_web_services_to_process', 'in', ['', False]), ('state', '=', 'draft')]}">
-                         <div>The invoice will be processed asynchronously by the following E-invoicing service :
+                         <div>The invoice will be processed automatically by the following E-invoicing service:
                             <field name="edi_web_services_to_process" class="oe_inline"/>
                          </div>
-                         <button name="button_process_edi_web_services" type="object" class="oe_link" string="Process now" />
+                         <button name="button_process_edi_web_services" type="object" class="btn btn-secondary" string="Process now" />
```

Comment:
Turning links like `Retry` `See errors` `Send `Process now` into buttons.
`Process now` will become `Send manually now` in the Italian EDI.